### ISSUE TITLE
Paper #454: DENAS: Automated Rule Generation by Knowledge Extraction from Neural Networks

### ISSUE THREAD

**Issue by kstolee** [researchart/fse20#83]
This issue is for discussion related to PR #43

Authors are applying for the *functional* badge

Author contact: @SeekingDream

**Comment by bigfatnoob** [comment]
## Authors:
|  Name | Email | GitHub ID | Corresponding author |
| ------- | ------| ---------- | --------------------- |
| Simin Chen | SXC180080@utdallas.edu | @DENAS-GLOBAL | ✔️ |
| Soroush Bateni |||
| Sampath Grandhi |||
| Xiaodi Li |||
| Cong Liu |||
| Wei Yang |||

**Comment by art-reviewer** [comment]
[Paper](https://github.com/SeekingDream/fse20/blob/890deb445a0c007d43483a456b9024f97cfff0eb/submissions/functional/denas/DENAS_FSE20.pdf)

**Comment by reviewer3** [comment]
This submission does not fulfill fundamental submission criteria (missing availability, license, DOI) nor does it adhere to the format laid out in the submission guidelines.

**Comment by anonReviewer10000** [comment]
This submission is missing some pieces before it can be assigned a badge, but several of the core elements are still there:

Documented: A brief README is provided which lists commands used to run the scripts.  The artifact is uploaded (https://github.com/researchart/fse20/tree/master/submissions/functional/denas) and dependencies are listed.

Consistent: The artifact is in line with the related paper.

Complete: The artifacts are uploaded as archive files, which limits usefulness a bit.  Ideally, the repository would be a navigable folder so files can be examined and documented individually prior to download.  Right now the repository is treated just as an upload directory.

Exercisable: Again, the format of the artifact makes this difficult.  Just uploading the rar files is not really in the spirit of the artifacts track, even if it is technically permitted.  It would be much more exercisable if the artifact were split up and the repository used as a real code repository.

**Comment by SeekingDream** [comment]
Dear reviewers,

we want to thank you for your valuable review #454. As per the review of @anonReviewer10000 and @reviewer3, we have done the following:

For available badge, we have archived our code repository at https://doi.org/10.5281/zenodo.3898178, and we apply the MIT license for our repository.
And we have unzipped the python script directory, now only the data and the model is stored in the *.zip files.

Thanks
Authors

**Comment by gousiosg** [comment]
@anonReviewer10000 and @reviewer3 would the additional information provided by the authors qualify them for the _available_ badge?

**Comment by anonReviewer10000** [comment]
@gousiosg Yes, the new DOI link and improved repository format are sufficient for 'available'.

**Comment by gousiosg** [comment]
Provisionally applied `available`. @reviewer3 let me know if you agree, so that we can close this.

**Comment by reviewer3** [comment]
With the changes, I am fine to grant available and then close this issue.
